python/meep.i

```diff
@@ -1813,6 +1814,15 @@ meep::structure *create_structure_and_set_materials(vector3 cell_size,
     meep_geom::fragment_stats::split_chunks_evenly = split_chunks_evenly;
     meep_geom::fragment_stats::init_libctl(_default_material, _ensure_periodicity,
                                            &gv, cell_size, center, &gobj_list);
+
+    if (output_chunk_costs) {
+         meep::volume thev = gv.surroundings();
+         std::vector<grid_volume> chunk_vols = meep::choose_chunkdivision(gv, thev, num_chunks, sym);
+         for (size_t i = 0; i < chunk_vols.size(); ++i)
+              master_printf("CHUNK:, %2zu, %f\n",i,chunk_vols[i].get_cost());
```

Comment:
You only need a few lines here to compute the minimum surface area of the `chunk_vols`, and output it.  (All of the old complications are no longer needed because there is only one chunk per process here.)

Any old code that is still needed by `visualize_chunks` can still be kept, of course.